File: Java Programing Basics/Nested Conditional Statements/src/FruitShop.java
import java.util.Scanner;

public class FruitShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String meal = scanner.nextLine();
        String Day = scanner.nextLine();
        double count = Double.parseDouble(scanner.nextLine());
        double totalPrice = -1;

        if (Day.equals("Monday") || Day.equals("Tuesday") || Day.equals("Wednesday") || Day.equals("Thursday") || Day.equals("Friday")) {
            double banana = 2.50;
            double apple = 1.20;
            double orange = 0.85;
            double grapefruit = 1.45;
            double kiwi = 2.70;
            double pineapple = 5.50;
            double grapes = 3.85;

            if (meal.equals("banana")) {
                totalPrice = banana * count;
            } else if (meal.equals("apple")) {
                totalPrice = apple * count;
            } else if (meal.equals("orange")) {
                totalPrice = orange * count;
            } else if (meal.equals("grapefruit")) {
                totalPrice = grapefruit * count;
            } else if (meal.equals("kiwi")) {
                totalPrice = kiwi * count;
            } else if (meal.equals("pineapple")) {
                totalPrice = pineapple * count;
            } else if (meal.equals("grapes")) {
                totalPrice = grapes * count;
            }


        } else if (Day.equals("Saturday") || Day.equals("Sunday")) {
            double banana = 2.70;
            double apple = 1.25;
            double orange = 0.90;
            double grapefruit = 1.60;
            double kiwi = 3.00;
            double pineapple = 5.60;
            double grapes = 4.20;

            if (meal.equals("banana")) {
                totalPrice = banana * count;
            } else if (meal.equals("apple")) {
                totalPrice = apple * count;
            } else if (meal.equals("orange")) {
                totalPrice = orange * count;
            } else if (meal.equals("grapefruit")) {
                totalPrice = grapefruit * count;
            } else if (meal.equals("kiwi")) {
                totalPrice = kiwi * count;
            } else if (meal.equals("pineapple")) {
                totalPrice = pineapple * count;
            } else if (meal.equals("grapes")) {
                totalPrice = grapes * count;
            }


        }
        if (totalPrice >= 0) {
            System.out.println(totalPrice);
        } else
            System.out.println("error");
    }
}

File: Python/Nested_Conditional_Statements/Fruit_Shop.py
fruit = input()
day = input()
count = float(input())

banana = 0
apple = 0
orange = 0
grapefruit = 0
kiwi = 0
pineapple = 0
grapes = 0
sum = 0


if day == "Saturday" or day == "Sunday":
    banana = 2.70
    apple = 1.25
    orange = 0.90
    grapefruit = 1.60
    kiwi = 3.00
    pineapple = 5.60
    grapes = 4.20

    if fruit == "banana":
        sum = banana * count
        print(f"{sum:.2f}")
    elif fruit == "apple":
        sum = apple * count
        print(f"{sum:.2f}")
    elif fruit == "orange":
        sum = orange * count
        print(f"{sum:.2f}")
    elif fruit == "grapefruit":
        sum = grapefruit * count
        print(f"{sum:.2f}")
    elif fruit == "kiwi":
        sum = kiwi * count
        print(f"{sum:.2f}")
    elif fruit == "pineapple":
        sum = pineapple * count
        print(f"{sum:.2f}")
    elif fruit == "grapes":
        sum = grapes * count
        print(f"{sum:.2f}")
    else:
        print("error")

elif day == "Monday" or day == "Tuesday" or day == "Wednesday" or day == "Thursday" or day == "Friday":

    banana = 2.50
    apple = 1.20
    orange = 0.85
    grapefruit = 1.45
    kiwi = 2.70
    pineapple = 5.50
    grapes = 3.85

    if fruit == "banana":
        sum = banana * count
        print(f"{sum:.2f}")
    elif fruit == "apple":
        sum = apple * count
        print(f"{sum:.2f}")
    elif fruit == "orange":
        sum = orange * count
        print(f"{sum:.2f}")
    elif fruit == "grapefruit":
        sum = grapefruit * count
        print(f"{sum:.2f}")
    elif fruit == "kiwi":
        sum = kiwi * count
        print(f"{sum:.2f}")
    elif fruit == "pineapple":
        sum = pineapple * count
        print(f"{sum:.2f}")
    elif fruit == "grapes":
        sum = grapes * count
        print(f"{sum:.2f}")
    else:
        print("error")
else:
    print("error")
